src/veh_interact.cpp

```diff
@@ -1172,9 +1176,10 @@ void veh_interact::do_tirechange()
         return;
     case LACK_TOOLS:
         fold_and_print(w_msg, 0, 1, msg_width - 2, c_ltgray,
-                       _("To change a wheel you need a <color_%1$s>wrench</color> and a <color_%2$s>jack</color>."),
+                       _("To change a wheel you need a <color_%1$s>wrench</color> and a <color_%2$s>jack</color>. Having strength %3$d or more allows to lift this car to change the tire."),
```

Comment:
The description is a bit misleading. How about something like "you need a wrench and either a jack or x str"?